src/app/user-page/user-page.component.html

```diff
@@ -121,6 +121,49 @@ <h4 matLine class="accent-1-dark mt-2 mb-0">{{ user?.username }}</h4>
         <app-recent-events></app-recent-events>
       </div>
     </mat-tab>
+    <mat-tab *ngIf="loggedInUserIsAdminOrCurator">
+      <ng-template mat-tab-label>
+        <div fxLayoutGap="0.5rem">
+          <span>Other Linked Accounts</span>
+        </div>
+      </ng-template>
+      <div class="p-2">
+        <div *ngFor="let token of tokens">
```

Comment:
Should handle the case for when there's no additional linked accounts otherwise it's just a blank tab